src/test/java/roomescape/member/application/MemberServiceTest.java

```diff
@@ -0,0 +1,77 @@
+package roomescape.member.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import roomescape.member.application.service.MemberService;
+import roomescape.member.presentation.dto.MemberResponse;
+import roomescape.member.presentation.dto.SignUpRequest;
+import roomescape.member.presentation.dto.SignUpResponse;
+import roomescape.member.presentation.dto.TokenRequest;
+
+@ActiveProfiles("test")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+public class MemberServiceTest {
```

Comment:
JUnitt5 부터는 테스트 클래스 및 메서드가 public 일 필요가 없습니다~
```suggestion
class MemberServiceTest {
```

Reply:
넵 수정 완료하였습니다!!

수정 commit: 752f12a